docs/group__eosc.html

```diff
@@ -85,7 +85,7 @@
iv class="line">      <span class="stringliteral">&quot;currency&quot;</span></div><div class="line">    ],</div><div class="line">    <span class="stringliteral">&quot;signatures&quot;</span>: [],</div><div class="line">    <span class="stringliteral">&quot;messages&quot;</span>: [{</div><div class="line">        <span class="stringliteral">&quot;code&quot;</span>: <span class="stringliteral">&quot;currency&quot;</span>,</div><div class="line">        <span class="stringliteral">&quot;type&quot;</span>: <span class="stringliteral">&quot;transfer&quot;</span>,</div><div class="line">        <span class="stringliteral">&quot;authorization&quot;</span>: [{</div><div class="line">            <span class="stringliteral">&quot;account&quot;</span>: <span class="stringliteral">&quot;currency&quot;</span>,</div><div class="line">            <span class="stringliteral">&quot;permission&quot;</span>: <span class="stringliteral">&quot;active&quot;</span></div><div class="line">          }</div><div class="line">        ],</div><div class="line">        <span class="stringliteral">&quot;data&quot;</span>: {</div><div class="line">          <span class="stringliteral">&quot;from&quot;</span>: <span class="stringliteral">&quot;currency&quot;</span>,</div><div class="line">          <span class="stringliteral">&quot;to&quot;</span>: <span class="stringliteral">&quot;tester&quot;</span>,</div><div class="line">          <span class="stringliteral">&quot;amount&quot;</span>: 50</div><div class="line">        },</div><div class="line">        <span class="stringliteral">&quot;hex_data&quot;</span>: <span class="stringliteral">&quot;a34a59dcc8000000b44c5a24000000003200000000000000&quot;</span></div><div class="line">      }</div><div class="line">    ],</div><div class="line">    <span class="stringliteral">&quot;output&quot;</span>: [{</div><div class="line">        <span class="stringliteral">&quot;notify&quot;</span>: [{</div><div class="line">            <span class="stringliteral">&quot;name&quot;</span>: <span class="stringliteral">&quot;tester&quot;</span>,</div><div class="line">            <span class="stringliteral">&quot;output&quot;</span>: {</div><div class="line">              <span class="stringliteral">&quot;notify&quot;</span>: [],</div><div class="line">              <span class="stringliteral">&quot;sync_transactions&quot;</span>: [],</div><div class="line">              <span class="stringliteral">&quot;async_transactions&quot;</span>: []</div><div class="line">            }</div><div class="line">          }</div><div class="line">        ],</div><div class="line">        <span class="stringliteral">&quot;sync_transactions&quot;</span>: [],</div><div class="line">        <span class="stringliteral">&quot;async_transactions&quot;</span>: []</div><div class="line">      }</div><div class="line">    ]</div><div class="line">  }</div><div class="line">}</div></div><!-- fragment --><p>The <code>exec</code> command takes the following arguments:</p>
```

Comment:
This is a modification to a generated file. This file should not be modified directly; rather, edits should be made to the source file at `contracts/eoslib/eosc.dox` and this file should be regenerated.